app/src/pages/inside/common/durationBlock/durationBlock.jsx

```diff
@@ -17,74 +17,20 @@
 import React, { Component, Fragment } from 'react';
 import classNames from 'classnames/bind';
 import Parser from 'html-react-parser';
-import { defineMessages, injectIntl } from 'react-intl';
+import { injectIntl } from 'react-intl';
 import PropTypes from 'prop-types';
 
 import { DottedPreloader } from 'components/preloaders/dottedPreloader';
 import { withTooltip } from 'components/main/tooltips/tooltip';
 import { getDuration, getApproximateTime } from 'common/utils';
 import ClockIcon from './img/clock-icon-inline.svg';
-import { DurationTooltip } from '../infoLine/parentInfo/duration/durationTooltip';
+import { DurationTooltip } from '../infoLine/parentInfo/duration/durationTooltip/durationTooltip';
```

Comment:
Looks like that `DurationTooltip` is a common integral part of the `DurationBlock`.
So it's mainly used with `DurationBlock` and additionally can be imported in some other places (f.e. in the `parentInfo/duration`).
So may be it's better to move it to the `DurationBlock` folder?

Reply:
Done.